java/client/src/org/openqa/selenium/WebElement.java

```diff
@@ -127,6 +127,7 @@
   /**
    * Determine whether or not this element is selected or not. This operation only applies to input
    * elements such as checkboxes, options in a select and radio buttons.
+   * This method supports the following elements: BUTTON, INPUT, OPTGROUP, OPTION, SELECT, TEXTAREA.
```

Comment:
The problem with HTML is they keep changing things. The actual definition of what can be checked to see if something is selected is defined in the [spec](https://w3c.github.io/webdriver/webdriver-spec.html#is-element-selected). It's probably best to either link to that, or to lift the meaningful section.

Further, the intention of the API is already described above the change you've made --- I'm not sure what the value is of keeping that statement with a more complete description being available.